src/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -35,6 +35,7 @@
  * Tactics are stateful, and use Primitives to implement their behaviour. They also
  * make heavy use of our Evaluation functions in order to help them make decisions.
  */
+template<class Tactic_FSM>
```

Comment:
```suggestion
template<class TacticFsm>
```
code style (treat acronyms as one word)